### PR TITLE
Pal 652 remove eureka beans

### PR DESCRIPTION
**Pull request by dev-958**

### CODE REVIEW

File: data-service-integration/src/test/resources/application.yaml [hunk 1]
```diff
@@ -47,6 +47,6 @@ feign:
 
 web:
   client:
-    audit-service: http://localhost:8088
+    audit-service: http://localhost:8098
```
Comment (ac74475) [inline data-service-integration/src/test/resources/application.yaml:50]: Why the change of port here?

Comment (dev-958) [reply data-service-integration/src/test/resources/application.yaml:50]: Test was barfing with "port in use" errors, but only on the cluster.

File: data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/DataComponentTest.java [hunk 2]
```diff
@@ -72,7 +69,6 @@
 @AutoConfigureMockMvc
```
Comment (m78233) [inline data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/DataComponentTest.java:69]: Don't think that the annotation @AutoConfiguredMocMvc is being used or needed.

File: data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/web/DataClientWrapper.java [hunk 3]
```diff
@@ -83,7 +84,8 @@ public DataClientWrapper(final DataClient client, final DataService service) {
      * @return              a {@link Boolean} value
      */
     public Boolean addSerialiser(final DataFlavour dataFlavour, final Serialiser<?> serialiser) {
-        return service.addSerialiser(dataFlavour, serialiser);
+        final AddSerialiserRequest request = new AddSerialiserRequest();
+        return service.addSerialiser(request.dataFlavour(dataFlavour).serialiser(serialiser));
```
Comment (ac74475) [inline data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/web/DataClientWrapper.java:88]: revert these changes

Comment (dev-958) [reply data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/web/DataClientWrapper.java]: reverted

File: data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/mock/AuditServiceMock.java [hunk 4]
```diff
@@ -31,7 +29,7 @@
 public class AuditServiceMock {
 
     public static WireMockRule getRule() {
-        return new WireMockRule(options().port(8088).notifier(new ConsoleNotifier(true)));
+        return new WireMockRule(options().port(8098).notifier(new ConsoleNotifier(true)));
```
Comment (dev930018) [inline data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/mock/AuditServiceMock.java:32]: Strictly follwing the pattern, shouldn't this be 8089?

Comment (dev-958) [reply data-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/data/mock/AuditServiceMock.java]: fat fingers, changed
